stories/components/peoplePicker.stories.js

```diff
@@ -29,11 +29,42 @@ export const groupId = () => html`
   <mgt-people-picker group-id="02bd9fd6-8f93-4758-87c3-1fb73740a315"></mgt-people-picker>
 `;
 
-export const DarkMode = () => html`
-  <mgt-people-picker theme="dark"></mgt-people-picker>
+export const theme = () => html`
```

Comment:
I like this story for the people picker. Should we also have one for the channel picker?

Reply:
Added. Also fixed the border color mixin orders to fix a little bug.